src/system/WatchableEventManagerSelect.h

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
Derived from `WatchableSocketSelect.h`; see https://gist.github.com/kpschoedel/9e64b03251925ab5b3df7bc61f0a0f78 for diff